core/src/main/java/com/sequenceiq/cloudbreak/service/rdsconfig/RdsConfigService.java

```diff
@@ -42,99 +42,73 @@
     private ClusterService clusterService;
 
     @Inject
-    private OrganizationService organizationService;
+    private RdsConnectionValidator rdsConnectionValidator;
 
-    public Set<RDSConfig> retrievePrivateRdsConfigs(IdentityUser user) {
-        return rdsConfigRepository.findForUser(user.getUserId());
+    public Set<RDSConfig> retrieveRdsConfigs() {
+        Organization organization = getOrganizationService().getDefaultOrganizationForCurrentUser();
+        return rdsConfigRepository.findAll(organization.getId());
     }
 
-    public RDSConfig getPrivateRdsConfig(String name, IdentityUser user) {
-        return Optional.ofNullable(rdsConfigRepository.findByNameInUser(name, user.getUserId()))
-                .orElseThrow(notFound("RDS configuration", name));
+    public RDSConfig getByName(String name) {
+        Organization organization = getOrganizationService().getDefaultOrganizationForCurrentUser();
+        return getByName(name, organization.getId());
     }
 
-    public RDSConfig getPublicRdsConfig(String name, IdentityUser user) {
-        return Optional.ofNullable(rdsConfigRepository.findByNameBasedOnAccount(name, user.getAccount(), user.getUserId()))
+    public RDSConfig getByName(String name, Long organizationId) {
```

Comment:
should be renamed to ```getByNameInOrganization``` or something like that. this method should be used everywhere basically